package.json

```diff
@@ -15,7 +15,7 @@
   },
   "scripts": {
     "pretest": "eslint --rulesdir=tools/eslint-rules lib test",
-    "test": "tap \"test/**/*.test.js\"",
+    "test": "tap test",
```

Comment:
Is it ok that this might load non-test js files?  (ones intended to be require()ed from elsewhere?)